charts/templates/statefulset.yaml

```diff
@@ -144,6 +144,31 @@ spec:
           {{- with .Values.resources }}
           resources: {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- if .Values.logCollection.enabled }}
+        - name: fluent-bit
+          image: {{ .Values.logCollection.image }}
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: datadir
+              mountPath: {{ .Values.persistence.mountPath }}
+            {{- if .Values.extraVolumeMounts }}
+              {{- toYaml .Values.extraVolumeMounts | nindent 12 }}
+            {{- end }}
+            - name: {{ include "marklogic.fullname" . }}-fb-config-map
+              mountPath: /fluent-bit/etc/
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          resources:
```

Comment:
The common practice for resource limit settings are like below:
in statefulSets:
resources: {{- toYaml .Values.logCollection.resources | nindent 12 }}
in the values. yaml:
requests:
  memory: "128Mi"
  cpu: "100m"
limits:
  memory: "128Mi"
  cpu: "100m"

Reply:
Change has been made